x-pack/plugins/security_solution/common/detection_engine/schemas/common/schemas.ts

```diff
@@ -453,6 +453,53 @@ export enum BulkAction {
   'export' = 'export',
   'delete' = 'delete',
   'duplicate' = 'duplicate',
+  'update' = 'update',
```

Comment:
I actually have been thinking to name it `edit`, but we already operate with name `update` when modifying rules(update APIs), so decided to go with `update` to be consistent.
Similarly, there are already defined bulk actions like `delete` that matches existing APIs(_bulk_delete).

Would having `edit` be preferable?

Reply:
Yeah, I think it would be preferable. This type of modification is different from the existing bulk update API (PUT semantics) and bulk patch API (PATCH semantics) - it implements idempotent adding and removal and could implement even more complicated logic in the future if we need it (off the top of my head - something like reset to initial params for modified prebuilt rules). In other words, bulk edit semantics != bulk update semantics. The reason I'm asking to not use the `update` name for this action is because I think it makes sense to reserve it for the simple bulk update logic if we make it a part of the bulk actions endpoint in the future.

cc @xcrzx @spong

Reply:
renamed as `edit`

Reply:
A little late and this has already been _updated_ (😉) , but +1 to `edit` semantics here. We'll probably want to touch base with @XavierM and the RAM (Rules/Alerts Mgmt) folks here soon as this is definitely something we'll want to fold into the existing alerting platform API (and so any feedback they may have would be useful at this point).

Reply:
@spong ++, we'll chat with Xavier on the future of Rule Management (including the APIs) in early January after all the holidays. Xavier is on PTO until the next year.